### ISSUE TITLE
Mint 19.2, updating starlabstheme-master, unable to open dark theme

### ISSUE THREAD

**Issue by dirtbirb** [StarLabsLtd/StarLabsTheme#12]
When using update manager to update starlabstheme-master, it fails with the info given below. Retrying doesn't help. I think I'm using the dark theme it complains about.

System:
StarLabs Labtop Mk III, purchased with Mint 19.1 and upgraded to 19.2 recently.

Error: 
E: /var/cache/apt/archives/starlabstheme-gtk_0.0.93_all.deb: unable to open '/usr/share/themes/StarLabs-Dark/gtk-3.0/assets/bullet-symbolic.svg.dpkg-new': No such file or directory

Full output:
(Reading database ... 347434 files and directories currently installed.)
Preparing to unpack .../00-mint-themes_1.8.2_all.deb ...
Unpacking mint-themes (1.8.2) over (1.8.1) ...
Preparing to unpack .../01-starlabstheme-backgrounds_0.0.93_all.deb ...
Unpacking starlabstheme-backgrounds (0.0.93) over (0.0.71) ...
Preparing to unpack .../02-starlabstheme-extensions_0.0.93_all.deb ...
Unpacking starlabstheme-extensions (0.0.93) over (0.0.71) ...
Preparing to unpack .../03-starlabstheme-font_0.0.93_all.deb ...
Unpacking starlabstheme-font (0.0.93) over (0.0.71) ...
Preparing to unpack .../04-starlabstheme-gnome-shell_0.0.93_all.deb ...
Unpacking starlabstheme-gnome-shell (0.0.93) over (0.0.71) ...
Preparing to unpack .../05-starlabstheme-grub_0.0.93_all.deb ...
Unpacking starlabstheme-grub (0.0.93) over (0.0.71) ...
Preparing to unpack .../06-starlabstheme-gtk_0.0.93_all.deb ...
Unpacking starlabstheme-gtk (0.0.93) over (0.0.71) ...
dpkg: error processing archive /tmp/apt-dpkg-install-bCX5QN/06-starlabstheme-gtk_0.0.93_all.deb (--unpack):
 unable to open '/usr/share/themes/StarLabs-Dark/gtk-3.0/assets/bullet-symbolic.svg.dpkg-new': No such file or directory
No apport report written because the error message indicates an issue on the local system
         Preparing to unpack .../07-starlabstheme-icons_0.0.93_all.deb ...
Unpacking starlabstheme-icons (0.0.93) over (0.0.71) ...
Preparing to unpack .../08-starlabstheme-plymouth_0.0.93_all.deb ...
Unpacking starlabstheme-plymouth (0.0.93) over (0.0.71) ...
Preparing to unpack .../09-starlabstheme-session_0.0.93_all.deb ...
Unpacking starlabstheme-session (0.0.93) over (0.0.71) ...
Preparing to unpack .../10-starlabstheme-sounds_0.0.93_all.deb ...
Unpacking starlabstheme-sounds (0.0.93) over (0.0.71) ...
Preparing to unpack .../11-starlabstheme_0.0.93_all.deb ...
Unpacking starlabstheme (0.0.93) over (0.0.71) ...
Errors were encountered while processing:
 /tmp/apt-dpkg-install-bCX5QN/06-starlabstheme-gtk_0.0.93_all.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)
A package failed to install.  Trying to recover:
Setting up starlabstheme-grub (0.0.93) ...
Installing new version of config file /etc/grub.d/47_starlabs ...
Sourcing file `/etc/default/grub'
Sourcing file `/etc/default/grub.d/50_linuxmint.cfg'
Generating grub configuration file ...
Found background image: wallpaper.png
Found linux image: /boot/vmlinuz-4.15.0-55-generic
Found initrd image: /boot/initrd.img-4.15.0-55-generic
Found linux image: /boot/vmlinuz-4.15.0-54-generic
Found initrd image: /boot/initrd.img-4.15.0-54-generic
Found linux image: /boot/vmlinuz-4.15.0-20-generic
Found initrd image: /boot/initrd.img-4.15.0-20-generic
File descriptor 9 (pipe:[102597]) leaked on lvs invocation. Parent PID 4995: /bin/sh
File descriptor 10 (pipe:[102597]) leaked on lvs invocation. Parent PID 4995: /bin/sh
File descriptor 11 (pipe:[96054]) leaked on lvs invocation. Parent PID 4995: /bin/sh
File descriptor 79 (/root/.synaptic/selections.proceed) leaked on lvs invocation. Parent PID 4995: /bin/sh
Adding boot menu entry for EFI firmware configuration
done
Setting up starlabstheme-font (0.0.93) ...
Setting up starlabstheme-gnome-shell (0.0.93) ...
Setting up starlabstheme-session (0.0.93) ...
Setting up starlabstheme-backgrounds (0.0.93) ...
Processing triggers for libglib2.0-0:amd64 (2.56.4-0ubuntu0.18.04.4) ...
Setting up starlabstheme-extensions (0.0.93) ...
Setting up mint-themes (1.8.2) ...
Setting up starlabstheme-sounds (0.0.93) ...
Setting up starlabstheme-plymouth (0.0.93) ...
update-initramfs: deferring update (trigger activated)
Setting up starlabstheme-icons (0.0.93) ...
Processing triggers for fontconfig (2.12.6-0ubuntu2) ...
Setting up starlabstheme (0.0.93) ...
Processing triggers for initramfs-tools (0.130ubuntu3.8) ...
update-initramfs: Generating /boot/initrd.img-4.15.0-55-generic

**Comment by Sean-StarLabs** [comment]
That's strange. Can you try opening a terminal and enter the below commands:
```
sudo apt clean
sudo apt install --reinstall starlabstheme-gtk
```

**Comment by BassmanBiff** [comment]
Same problem, output below (I don't think it's any different):

Reading package lists... Done
Building dependency tree
Reading state information... Done
The following packages will be upgraded:
  starlabstheme-gtk
1 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Need to get 138 kB of archives.
After this operation, 60.4 kB of additional disk space will be used.
Get:1 http://ppa.launchpad.net/starlabs/ppa/ubuntu bionic/main amd64
starlabstheme-gtk all 0.0.93 [138 kB]
Fetched 138 kB in 1s (146 kB/s)
(Reading database ... 352919 files and directories currently installed.)
Preparing to unpack .../starlabstheme-gtk_0.0.93_all.deb ...
Unpacking starlabstheme-gtk (0.0.93) over (0.0.71) ...
dpkg: error processing archive
/var/cache/apt/archives/starlabstheme-gtk_0.0.93_all.deb (--unpack):
 unable to open
'/usr/share/themes/StarLabs-Dark/gtk-3.0/assets/bullet-symbolic.svg.dpkg-new':
No such file or directory
No apport report written because the error message indicates an issue on
the local system
         Errors were encountered while processing:
 /var/cache/apt/archives/starlabstheme-gtk_0.0.93_all.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)

**Comment by Sean-StarLabs** [comment]
Okay, thanks. Can you try this:
```
sudo apt remove starlabs-labtop-mk-iii
sudo apt autoremove
sudo apt clean
sudo apt install starlabs-labtop-mk-iii
```

**Comment by BassmanBiff** [comment]
That worked ok, but didn't impact the themes package. I tried apt update
and apt upgrade afterward and got the same error.

**Comment by Sean-StarLabs** [comment]
Ah okay. Updated to specifically target the themes:
```
sudo apt remove starlabs-labtop-mk-iii starlabstheme starlabstheme-*
sudo apt autoremove
sudo apt clean
sudo apt install starlabs-labtop-mk-iii
```

**Comment by BassmanBiff** [comment]
That seems to have done it! Dunno how that got broken. Thanks for the help,
hope the feedback was useful.

**Comment by Sean-StarLabs** [comment]
Awesome :)

It was indeed, thank you. I'll start digging now to figure out what happened!

**Comment by B7ADE-RUNNER** [comment]
Hello, I am having a similar issue with this package starlabstheme-master

E: /var/cache/apt/archives/starlabstheme-gtk_0.0.96_all.deb: unable to open '/usr/share/themes/StarLabs-Dark/gtk-3.0/assets/bullet-symbolic.svg.dpkg-new': No such file or directory

I tried what was suggested here, but it does not show. It installed a wealth of other themes, but all the Dark versions of the themes do not show. Even though those files are there when I look for them.

**Comment by Sean-StarLabs** [comment]
Can you try running:
```
sudo find /usr/share/themes/StarLabs-Dark/gtk-3.0/ -type l -delete
```
And then try installing again?

**Comment by B7ADE-RUNNER** [comment]
It reinstalled the assets file, but StarLabs-Dark isn't showing up in any of the themes. Also, this package starlabstheme-master is no longer showing in Update Manager.

**Comment by Sean-StarLabs** [comment]
Sorry, not sure what you mean. How are you installing it and on what distro?

**Comment by B7ADE-RUNNER** [comment]
Like the OP I upgraded to Linux Mint 19.2, and the starlabstheme-master package showed up in the update manager for an update. After I tried to upgrade it, it displayed that same error message. I deleted the file from the gtk-3.0 folder using the terminal command you suggested. I installed using 

sudo apt install starlabs-labtop-mk-iii starlabstheme starlabstheme-*

and the assets file within the gtk-3.0 directory showed up again, and contained all the images.

It is not showing up in my system preferences. I rather enjoyed the StarLabs-Dark cursor theme. None of the dark themes, which are present in /usr/share/themes/ are showing up in the system preferences.
